src/TheLow.sol

```diff
@@ -24,7 +24,7 @@ contract TheLow is ERC721, Owned {
         string image_cid;
         string animation_cid;
         string animation_hash;
-        uint8 portion;  // How many tokenIds, out of 222, of this type will be created
+        uint16 portion;  // How many tokenIds, out of 222, of this type will be created
```

Comment:
Why not uint8, and it goes up to 255?

Reply:
Because once it was changed to "proportion" to handle less-than-sellout, it becomes the denominator in (total supply / portion = number).  I needed 2 decimal places to make the math work right (e.g. 222 / 20.19 = 75) so uint8 wasn't enough anymore.

I should, however, fix the comment to reflect that.